Tests/GRPCNIOTransportHTTP2Tests/HTTP2TransportTLSEnabledTests.swift

```diff
@@ -154,8 +154,7 @@ struct HTTP2TransportTLSEnabledTests {
 
   @Test(
     "Error is surfaced when server fails client verification",
-    arguments: TransportKind.supported,
-    TransportKind.supported
+    arguments: [TransportKind.posix], [TransportKind.posix]
```

Comment:
We should be using `.supported` here, right?

Reply:
Oops, yes